raibove/Task4/my_timer.js

```diff
@@ -0,0 +1,45 @@
+
+	var secs = 0;
+	var timeout;
+	var mins = 0;
```

Comment:
Instead of using `var`, use `const`(or, if the value needs to be changed on a later stage, use `let`).
> `let` and `const` behave exactly the same way in terms of Lexical Environments. But `var` is a very different beast, that originates from very old times. It’s generally not used in modern scripts, but still lurks in the old ones. [source](https://javascript.info/var)
```suggestion
	let secs = 0;
	let timeout;
	let mins = 0;
```